run.py

```diff
@@ -103,12 +103,14 @@ def main(args):
                 )
 
                 feature_names = model.get_feature_names()
-                for i, (importance, index, is_positive) in enumerate(
-                    importance["importances"]
-                ):
-                    print(
-                        f'{i + 1}. \'{feature_names[int(index)]}\' ({"+" if (is_positive) else "-"}{importance})'
-                    )
+
+                # shap values of top features for the predicted class
+                pred_class_index = int(np.where(probas == np.amax(probas))[1])
+
+                model.print_feature_importances(
+                    importance["importances"], feature_names, pred_class_index
```

Comment:
@marco-c i need help here , when i call this (even with pred_class = pred_class_index), pred_class is being treated as False(the default values) :(

Reply:
i got it.